src/app/main.cpp

```diff
@@ -28,21 +28,36 @@ static CapturePtr create_capture(Context context) {
   auto monitor = sc::GetMonitors().front();
```

Comment:
why do you use sc::GetMonitors().front() here. it doesn't work with multi monitors at all. you should check monitors before logging any information about monitor name, width, etc.